monai/losses/boundary_enhancement_loss.py

```diff
@@ -0,0 +1,76 @@
+import monai
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class BoundaryLoss(nn.Module):
+    def __init__(self, output_classes, device):
+        super().__init__()
+
+        self.output_classes = output_classes
+
+        self.conv1 = nn.Conv3d(
```

Comment:
this could be extended to support 2d, could you also add some unit tests? thanks!